jina/docker/hubio.py

```diff
@@ -344,8 +344,9 @@ def build(self) -> Dict:
                     p_names = []
                     try:
                         is_build_success = False
-                        p_names = self._test_build(image)
-                        is_build_success = True
+                        p_names, failed_test_levels = self._test_build(image)
+                        if BuildTestLevel.POD_DOCKER or BuildTestLevel.EXECUTOR in failed_test_levels:
```

Comment:
this is wrong, the logic should be negative, is more about which errors we do not allow rather than what we accept

Reply:
Now if every level fails, we will consider it a success